src/Microsoft.ML.HalLearners/HalLearnersCatalog.cs

```diff
@@ -44,11 +65,31 @@ public static class HalLearnersCatalog
         public static SymSgdClassificationTrainer SymbolicStochasticGradientDescent(this BinaryClassificationContext.BinaryClassificationTrainers ctx,
             string labelColumn = DefaultColumnNames.Label,
             string featureColumn = DefaultColumnNames.Features,
-            Action<SymSgdClassificationTrainer.Arguments> advancedSettings = null)
+            Action<SymSgdClassificationTrainer.Options> advancedSettings = null)
```

Comment:
This argument is unused. Drop it?

Reply:
thanks

---
In reply to: [248767626](https://github.com/dotnet/machinelearning/pull/2170#discussion_r248767626) [](ancestors = 248767626)